bridge/gateway.bridge/pom.xml

```diff
@@ -36,6 +36,20 @@
     <build>
         <plugins>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <configuration>
+                    <archive>
+                        <manifestEntries>
+                            <Implementation-Title>${project.name}</Implementation-Title>
+                            <Implementation-Version>${project.version}</Implementation-Version>
+                            <Jar-Name>${artifactId}</Jar-Name>
```

Comment:
Why not use the {artifactId} for the Kaazing-Product category as well? You may then be able to use inheritance to minimize the number of plugins that you need, i.e. just put this in the top level pom (maybe, not sure how inheritance works it the submodule is not of packaging jar